package.json

```diff
@@ -81,7 +81,7 @@
     "netjet": "1.3.0",
     "nodemailer": "0.7.1",
     "oauth2orize": "1.11.0",
-    "oembed-parser": "1.1.1",
+    "oembed-parser": "https://github.com/kevinansfield/oembed-parser.git#ghost",
```

Comment:
I guess it was your intension to point to a branch?

I usually point to a specific commit, because you might forget about it and introduce an unstable commit on this branch.

Reply:
No i don't prefer :) Up to you. Just wanted to ask 😎